lib/clean-host.js

```diff
@@ -17,16 +17,20 @@ var URL = require('url');
 var hasPrefixRE = /^(([a-z][a-z0-9+.-]*)?:)?\/\//;
 var invalidHostnameChars = /[^A-Za-z0-9.-]/;
 
-function trim(value) {
-  return String(value).replace(/(^\s+|\s+$)/g, '');
+function ltrim(value) {
```

Comment:
Thinking about it, we could also just use the plain old [`String.prototype.trim`](http://kangax.github.io/compat-table/es5/#test-String_properties_and_methods_String.prototype.trim_a_href=_https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim_title=_MDN_documentation_img_src=_../mdn.png_alt=_MDN_(Mozilla_Development_Network)_logo_width=_15_height=_13_/_/a_nbsp;) as it is supported since ECMAScript 5.0.

I'll make the change prior to publish the new release 👍